controls/V-71891.rb

```diff
@@ -60,45 +44,33 @@
 
 If the \"lock-enabled\" setting is missing or is not set to \"true\", this is a
 finding."
-  tag "fix": "Configure the operating system to enable a user's session lock until
-that user re-establishes access using established identification and authentication
-procedures.
+  tag "fix": "Configure the operating system to enable a user's session lock
+until that user re-establishes access using established identification and
+authentication procedures.
 
-Create a database to contain the system-wide screensaver settings (if it does not
-already exist) with the following command:
+Create a database to contain the system-wide screensaver settings (if it does
+not already exist) with the following command:
 
 # touch /etc/dconf/db/local.d/00-screensaver
 
-Edit “org/gnome/desktop/session” and add or update the following lines:
-
-# Set the lock time out to 900 seconds before the session is considered idle
-idle-delay=uint32 900
-
 Edit \"org/gnome/desktop/screensaver\" and add or update the following lines:
 
 # Set this to true to lock the screen when the screensaver activates
 lock-enabled=true
-# Set the lock timeout to 180 seconds after the screensaver has been activated
-lock-delay=uint32 180
-
-You must include the \"uint32\" along with the integer key values as shown.
-
-Override the user's setting and prevent the user from changing it by editing
-“/etc/dconf/db/local.d/locks/screensaver” and adding or updating the following lines:
-
-# Lock desktop screensaver settings
-/org/gnome/desktop/session/idle-delay
-/org/gnome/desktop/screensaver/lock-enabled
-/org/gnome/desktop/screensaver/lock-delay
 
 Update the system databases:
 
 # dconf update
 
-Users must log out and back in again before the system-wide settings take effect."
-
+Users must log out and back in again before the system-wide settings take
+effect."
+  tag "fix_id": "F-78243r7_fix"
   describe parse_config_file('/etc/dconf/db/local.d/00-screensaver') do
```

Comment:
This file path is arbitrary. The `gsettings` tool should be used to gather these values.